### PR TITLE
Ensure credentials are not being stored or captured

### LINKED ISSUE #52: Confirm if captured passwords are stored in plain text
# 💡 Summary #

Confirm if captured passwords are stored in plain text

## Motivation and context ##

Per Dave, "confirm if captured passwords are still stored in plain text. If so, this comment should be updated from 0.9.0. to whatever the current Gophish version is." 
_Originally posted by @dav3r in https://github.com/cisagov/gophish-tools/pull/50#discussion_r732112496_

## Acceptance criteria ##

How do we know when this work is done?

- [ ] Passwords are not stored in plain text
- [ ] Gophish version is updated in the README file

### PR DESCRIPTION
**Pull request by JCantu248**
#  #

## 🗣 Description ##

Minor revision to configure all campaigns to not allow either credentials or passwords to be captured by Gophish.

## 💭 Motivation and context ##

Discussing with Kelly Theile and the PCA team, we discussed if passwords are being stored in plaintext (#52) and we were informed that we are not allowed to receive any credentials in any form (plaintext or encrypted) for legal reasons. Credentials must be dropped by the landing pages. This change ensures that no credential or password data is captured. 

In addition, setting the capture credential and password  boolean values was done in one place, before being repeated on each branch of execution. 

Closes #52 

## 🧪 Testing ##

This was tested by bringing the code changes into our LiPCA process and checking that the campaigns created afterwards were no longer configured to capture submitted data. After setting capture credentials to false, the checkbox in the Gophish GUI for capture submitted data is no longer set. 

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: src/_version.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 """This file defines the version of this project."""
-__version__ = "0.0.8"
+__version__ = "0.0.10"
```
Comment (dav3r) [inline src/_version.py]: What happened to version `0.0.9`?  

On a related note, do you have an idea of what has to happen for this project to reach version `1.0.0`?

**Review by dav3r — APPROVED** [review]
This looks good to me, though please see my questions about versions.

File: src/_version.py (hunk 1 above)
Comment (JCantu248) [reply src/_version.py]: I think somewhere along the way, I must have screwed up with the bump_version tool and making a pre-release. I think I might have already used the v0.0.9 tag accidentally.

Comment (JCantu248) [reply src/_version.py]: Also regarding a v.1, I think we have some Sprint planning we are doing currently for our Live PCA project, and that may define what v.1 of Gophish-tools would look like.

Comment (dav3r) [reply src/_version.py]: According to the rules at [semver.org](https://semver.org/), this should probably be a minor release, meaning you could bump it to version `0.1.0` if you wanted.

**Comment by dav3r** [comment]
You should also merge in the latest changes from `develop` to ensure there are no conflicts.

File: src/_version.py (hunk 1 above)
Comment (JCantu248) [reply src/_version.py]: I bumped it to v0.1.0 in 4a392778cc9f97d4950443b2a11aa44ca6d64237 

Additionally, I merged the latest changes from develop into this feature branch.

**Review by dav3r — APPROVED** [review]

**Review by jsf9k — APPROVED** [review]
:+1:

**Pull request merged**